ceph-volume-test/config/definitions/ceph-volume-test.yml

```diff
@@ -33,186 +33,412 @@
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos8-filestore-create
+                    SUBCOMMAND=lvm
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos8-bluestore-dmcrypt
+                    SUBCOMMAND=lvm
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos8-filestore-dmcrypt
+                    SUBCOMMAND=lvm
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos7-bluestore-prepare_activate
+                    SUBCOMMAND=lvm
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos7-filestore-prepare_activate
+                    SUBCOMMAND=lvm
           - multijob:
               name: 'testing ceph-volume simple'
               condition: SUCCESSFUL
               projects:
                 - name: ceph-volume-scenario
                   current-parameters: true
                   predefined-parameters: |
-                    SCENARIO=xenial-bluestore-activate
+                    SCENARIO=centos8-bluestore-activate
                     SUBCOMMAND=simple
                 - name: ceph-volume-scenario
                   current-parameters: true
                   predefined-parameters: |
-                    SCENARIO=xenial-filestore-activate
+                    SCENARIO=centos8-filestore-activate
                     SUBCOMMAND=simple
                 - name: ceph-volume-scenario
                   current-parameters: true
                   predefined-parameters: |
-                    SCENARIO=centos7-bluestore-activate
+                    SCENARIO=centos8-bluestore-dmcrypt_luks
                     SUBCOMMAND=simple
                 - name: ceph-volume-scenario
                   current-parameters: true
                   predefined-parameters: |
-                    SCENARIO=centos7-filestore-activate
+                    SCENARIO=centos8-filestore-dmcrypt_luks
                     SUBCOMMAND=simple
                 - name: ceph-volume-scenario
                   current-parameters: true
                   predefined-parameters: |
-                    SCENARIO=xenial-bluestore-dmcrypt_luks
+                    SCENARIO=centos8-bluestore-dmcrypt_plain
                     SUBCOMMAND=simple
                 - name: ceph-volume-scenario
                   current-parameters: true
                   predefined-parameters: |
-                    SCENARIO=xenial-filestore-dmcrypt_luks
+                    SCENARIO=centos8-filestore-dmcrypt_plain
                     SUBCOMMAND=simple
```

Comment:
simple on makes sense on luminous and mimic, so iiuc this can be dropped from this step.